task-ballerina/src/task/listener.bal

```diff
@@ -177,4 +182,10 @@ isolated function validateConfiguration(TimerConfiguration|AppointmentConfigurat
     } else if (!(misfirePolicy is AppointmentMisfirePolicy)) {
         panic ListenerError("Wrong misfire policy has given for the appointment task.");
     }
+    if (threadConfiguration.threadCount < 1) {
```

Comment:
Seems this validation is redundant since this is captured in the next validation.